data/human/pirates/pirate intro.txt

```diff
@@ -0,0 +1,734 @@
+		`	"Slavery? You're a monster."`
+			goto slaveryspeech
+	apply
+		set terribleperson
+	`	"I'm glad we see eye to eye. Protection racket, landlord, slavemaster; whatever people want to call it, it's simply the natural order of things that some are strong, and some are weak. The strong thrive because they put to better use that which the weak produce. And you? I know which of the two you are now. I'll put your name into the ears of others like us."`
+	#the ring token is for integration into the pirate campaign
+		#goto ring
+		goto outlawapproval
+	label slaveryspeech
+	`	"Don't be stupid. We're in the same business, you and I - the only reason we're having this conversation at all is because you saw fit to force others to pay you, render unto you the fruit of their sweat and tears, without anything in return. You're a slaver as much as I; I'm simply more honest about it. You'll grow to accept that. Pirates who think they can survive - or more amusingly, thrive - and still hold onto their precious morals; they're not long for this galaxy."`
+	`	The Corsair's lips curl into a decidedly displeased scowl, but he drowns any words in liquor before they might threaten to spill forth.`
+	`	"Now. Don't be a fool, and remember my words." The Capo relaxes, stretching back in his seat. He's regarding you cooly, contemplatively. "But I'm a man of honor; you did as you were instructed, and so I'll fulfil my end of the bargain."`
+		goto tributeskip
+	label defender
+	choice
+		`	(You address the Capo.) "I conquered Darkstone, defeating the gang that once claimed it. When others came to enslave my subjects I defeated them as well."`
+	`	"You little shit. I gave you a job to do, an opportunity, and you repay me by killing my men? Did you think you could just muscle in to the trade without people backing you up?" Cold rage burns within the cyborg's eyes, his expression hard to read given lack of lips and most facial muscles.`
+	choice
+		`	"They got what was coming to them. That territory was mine by right of conquest, and by trespassing they signed their own death warrants. You respect my turf and I'll respect you, understood?"`
+			goto grudgingrespect
+		`	"I didn't know those were your men, I simply saw raiders coming in to steal from my new tributaries and I didn't want that to happen. I'm sorry."`
+			goto weak
+		`	"Damn slavers. If I'd known that's what you'd intended all along I never would have gone along with your plan. You disgust me."`
+	`	The Capo fixes you with a hateful gaze. "Don't be stupid. We're in the same business, you and I - the only reason we're having this conversation at all is because you saw fit to force others to pay you, render unto you the fruit of their sweat and tears, without anything in return. You're a slaver as much as I; I'm simply more honest about it. You'll grow to accept that, pirates that think they can survive - or more amusingly, thrive - and still hold onto their precious morals, they're not long for this galaxy."`
```

Comment:
It was either this or apply a condition in the lead-up to line 172. The followup lines to this blurb are necessarily different because they come from different start conditions.

Reply:
I understand. I'd slightly prefer a condition and a common label.

Reply:
I don't think I can give these a common label and still have the differences be within the same paragraph, you can only do labels after a new line.

Reply:
Okay, looking this over I could probably make a weave of applying conditions and then labels that redirect once you have the right condition, but that's massively more complicated than just having two different flows of conversation which happen to include three quarters of a paragraph in common.

Reply:
(That is to say, I'd just very much rather not and leave it as-is)

Reply:
I definitely get wanted the text to be as compact as possible. But yeah, these are not the same line and can't be shared between dialog trees without throwing an ugly line break in there.

Reply:
ok.